src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
The `Ball` class should have an access modifier, preferably `public` to ensure that it can be accessed from other packages if necessary.

Reply:
The `Ball` class already has the `public` access modifier.